README.md

```diff
@@ -214,7 +222,7 @@ name=SecureDrop Workstation Qubes initial install bootstrap
 
 3. Download the RPM package
 ```
-[user@work ~]$ sudo dnf download securedrop-workstation-dom0-config
+[user@work ~]$ dnf download securedrop-workstation-dom0-config
 ```
 
 The RPM file will be downloaded to your current working directory.
```

Comment:
probably worth fixing this while you're updating the prod install instructions: I think the step below (step 4) also needs a modification: `[dom0]$ qvm-run --pass-io work '/home/user/securedrop-workstation-dom0-config-x.y.z-1.fc25.noarch.rpm' > securedrop-workstation.rpm` -> `[dom0]$ qvm-run --pass-io work 'cat /home/user/securedrop-workstation-dom0-config-x.y.z-1.fc25.noarch.rpm' > securedrop-workstation.rpm`

Reply:
good catch, fixed